app.json

```diff
@@ -4,7 +4,7 @@
     "slug": "liquid",
     "version": "1.0.0",
     "orientation": "portrait",
-    "icon": "./assets/images/icon.png",
+    "icon": "./assets/images/liquid-icon.png",
```

Comment:
@MeisterJustice you need to run the prebuild command to update ios and android folders, no?

Reply:
what's the command please? i believe i did @renanmav

Reply:
@MeisterJustice I don't see changes in folders `ios` and `android` in the diff.

Commands are:

```bash
npx expo prebuild -p ios
npx expo prebuild -p android
```

Reply:
thank you

Reply:
done @renanmav